projects/step-core/src/lib/pipes/plan-url.pipe.ts

```diff
@@ -0,0 +1,19 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { Plan } from '../client/step-client-module';
+
+@Pipe({
+  name: 'planUrl',
+})
+export class PlanUrlPipe implements PipeTransform {
+  static transform(idOrPlan?: string | Plan): string {
+    if (!idOrPlan) {
+      return '';
+    }
+    const id = typeof idOrPlan === 'string' ? idOrPlan : idOrPlan.id;
```

Comment:
```suggestion
    const id = (typeof idOrPlan === 'string' ? idOrPlan : idOrPlan.id);
```

I prefer brackets for readability